app/uk/gov/hmrc/emcstfe/config/AppConfig.scala

```diff
@@ -47,6 +47,10 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig, configuration: Configu
 
   def cancelAMovementUserAnswersReplaceIndexes(): Boolean = configuration.get[Boolean]("mongodb.cancelAMovementUserAnswers.replaceIndexes")
 
+  def changeDestinationUserAnswersTTL(): Duration = Duration(configuration.get[String]("mongodb.changeDestinationUserAnswers.TTL"))
+
+  def changeDestinationUserAnswersReplaceIndexes(): Boolean = configuration.get[Boolean]("mongodb.changeDestinationUserAnswers.replaceIndexes")
```

Comment:
Will need an `app-config-production` entry for these, to ensure that `replaceIndexes` is set to `false` by default when we deploy to Production and so it's clear to see what the TTL is per collection in Prod.

Add to this:
https://github.com/hmrc/app-config-production/blob/main/emcs-tfe.yaml#L47-L58